Build failure: cemu-2.0-82 / 2.0-85

### Steps To Reproduce

Steps to reproduce the behavior:
1. build cemu-2.0-82 or 2.0-85

### Build log

```
@nix { "action": "setPhase", "phase": "unpackPhase" }
Running phase: unpackPhase
unpacking source archive /nix/store/rvqwapm6fniqrhv0p4qs5q9f6y4p858l-source
source root is source
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: patchPhase
applying patch /nix/store/j864w8ixh8yhig6ly4357aybnxfvx3zi-cmakelists.patch
patching file CMakeLists.txt
Hunk #1 succeeded at 125 (offset 33 lines).
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "configurePhase" }
Running phase: configurePhase
substituteStream(): WARNING: '--replace' is deprecated, use --replace-{fail,warn,quiet}. (file 'src/Common/version.h')
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/m2mkkndifsgk5y1m5gzycfaz5gj9gvhv-cemu-2.0-82/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/m2mkkndifsgk5y1m5gzycfaz5gj9gvhv-cemu-2.0-82/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/m2mkkndifsgk5y1m5gzycfaz5gj9gvhv-cemu-2.0-82/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/m2mkkndifsgk5y1m5gzycfaz5gj9gvhv-cemu-2.0-82/share/doc/Cemu -DCMAKE_INSTALL_INFODIR=/nix/store/m2mkkndifsgk5y1m5gzycfaz5gj9gvhv-cemu-2.0-82/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/m2mkkndifsgk5y1m5gzycfaz5gj9gvhv-cemu-2.0-82/share/man -DCMAKE_INSTALL_OLDINCLUDEDIR=/nix/store/m2mkkndifsgk5y1m5gzycfaz5gj9gvhv-cemu-2.0-82/include -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/m2mkkndifsgk5y1m5gzycfaz5gj9gvhv-cemu-2.0-82/include -DCMAKE_INSTALL_SBINDIR=/nix/store/m2mkkndifsgk5y1m5gzycfaz5gj9gvhv-cemu-2.0-82/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/m2mkkndifsgk5y1m5gzycfaz5gj9gvhv-cemu-2.0-82/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/m2mkkndifsgk5y1m5gzycfaz5gj9gvhv-cemu-2.0-82/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_OSX_SYSROOT= -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/9bv7dcvmfcjnmg5mnqwqlq2wxfn8d7yi-gcc-wrapper-13.2.0/bin/strip -DCMAKE_RANLIB=/nix/store/9bv7dcvmfcjnmg5mnqwqlq2wxfn8d7yi-gcc-wrapper-13.2.0/bin/ranlib -DCMAKE_AR=/nix/store/9bv7dcvmfcjnmg5mnqwqlq2wxfn8d7yi-gcc-wrapper-13.2.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/m2mkkndifsgk5y1m5gzycfaz5gj9gvhv-cemu-2.0-82 -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG -DENABLE_VCPKG=OFF -DENABLE_FERAL_GAMEMODE=ON -DPORTABLE=OFF 
-- The C compiler identification is GNU 13.2.0
-- The CXX compiler identification is GNU 13.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/9bv7dcvmfcjnmg5mnqwqlq2wxfn8d7yi-gcc-wrapper-13.2.0/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/9bv7dcvmfcjnmg5mnqwqlq2wxfn8d7yi-gcc-wrapper-13.2.0/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Found CURL: /nix/store/36kr91f8s509z2r8hqvpayw5bqyqgyzr-curl-8.7.1/lib/libcurl.so (found version "8.7.1")
-- RapidJSON found. Headers: /nix/store/ywz4vypwn19nnlcsmkdndiz05wg58l0m-rapidjson-1.1.0/include
-- Found Boost: /nix/store/ywac4cc4ls0nckfpbc5w2wfrd27sxrlh-boost-1.81.0-dev/lib/cmake/Boost-1.81.0/BoostConfig.cmake (found version "1.81.0") found components: program_options filesystem nowide
-- Found ZLIB: /nix/store/lv6nackqis28gg7l2ic43f6nk52hb39g-zlib-1.3.1/lib/libz.so (found version "1.3.1")
-- Found zstd: /nix/store/9ynakmwsf7nqhlv3phl05yybc5ad754l-zstd-1.5.6-dev/lib/cmake/zstd/zstdConfig.cmake (found version "1.5.6")
-- Found OpenSSL: /nix/store/gp504m4dvw5k2pdx6pccf1km79fkcwgf-openssl-3.0.13/lib/libcrypto.so (found version "3.0.13") found components: Crypto SSL
[0mCMake Deprecation Warning at /nix/store/wzpzgsca98qwpdc54qcqxy5bz9i70fpn-glm-0.9.9.8/lib/cmake/glm/glmConfig.cmake:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.
Call Stack (most recent call first):
  CMakeLists.txt:133 (find_package)

[0m
[0mCMake Deprecation Warning at /nix/store/wzpzgsca98qwpdc54qcqxy5bz9i70fpn-glm-0.9.9.8/lib/cmake/glm/glmConfig.cmake:2 (cmake_policy):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.
Call Stack (most recent call first):
  CMakeLists.txt:133 (find_package)

[0m
-- Found PNG: /nix/store/55kivg8damd3y2c5k7286nvckw8pb2xf-libpng-apng-1.6.43/lib/libpng.so (found version "1.6.43")
-- Found X11: /nix/store/ny34a9migrcin09zwgbkr0ymx72mvlfg-xorgproto-2024.1/include
-- Looking for XOpenDisplay in /nix/store/x9fw7rbdb34gq0f8q750kw344lbv9nk1-libX11-1.8.9/lib/libX11.so;/nix/store/gbjygp4wz7b5rgayckmqfc00hy34dqfn-libXext-1.3.6/lib/libXext.so
-- Looking for XOpenDisplay in /nix/store/x9fw7rbdb34gq0f8q750kw344lbv9nk1-libX11-1.8.9/lib/libX11.so;/nix/store/gbjygp4wz7b5rgayckmqfc00hy34dqfn-libXext-1.3.6/lib/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Looking for IceConnectionNumber in ICE
-- Looking for IceConnectionNumber in ICE - found
[33mCMake Warning (dev) at cmake/ECMFindModuleHelpers.cmake:113 (message):
  Your project should require at least CMake 3.16.0 to use FindWayland.cmake
Call Stack (most recent call first):
  cmake/FindWayland.cmake:61 (ecm_find_package_version_check)
  CMakeLists.txt:145 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.
[0m
-- Found Wayland_Client: /nix/store/7rnc20sb0h5zxqm5vims6mvi6gr3iy27-wayland-1.22.0/lib/libwayland-client.so (found version "1.22.0")
-- Found Wayland: /nix/store/7rnc20sb0h5zxqm5vims6mvi6gr3iy27-wayland-1.22.0/lib/libwayland-client.so (found version "1.22.0") found components: Client
[33mCMake Warning (dev) at cmake/ECMFindModuleHelpers.cmake:113 (message):
  Your project should require at least CMake 3.16.0 to use
  FindWaylandScanner.cmake
Call Stack (most recent call first):
  cmake/FindWaylandScanner.cmake:68 (ecm_find_package_version_check)
  CMakeLists.txt:146 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.
[0m
-- Found WaylandScanner: /nix/store/zkxwpiayw93k1lcmym4gsi76i4irq4d9-wayland-1.22.0-bin/bin/wayland-scanner
-- Found WaylandProtocols: //nix/store/g3pc2hibffh9wcbv8rjxhvp51r3lk6zc-wayland-protocols-1.35/share/wayland-protocols (found suitable version "1.35", minimum required is "1.15")
-- Found PkgConfig: /nix/store/zdvrzlvzbn9ymb0z8na50w995j8np16z-pkg-config-wrapper-0.29.2/bin/pkg-config (found version "0.29.2")
-- Checking for one of the modules 'gtk+-3.0'
Package libpcre2-8 was not found in the pkg-config search path.
Perhaps you should add the directory containing `libpcre2-8.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libpcre2-8', required by 'glib-2.0', not found
Package libpcre2-8 was not found in the pkg-config search path.
Perhaps you should add the directory containing `libpcre2-8.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libpcre2-8', required by 'glib-2.0', not found
Package libpcre2-8 was not found in the pkg-config search path.
Perhaps you should add the directory containing `libpcre2-8.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libpcre2-8', required by 'glib-2.0', not found
Package libpcre2-8 was not found in the pkg-config search path.
Perhaps you should add the directory containing `libpcre2-8.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libpcre2-8', required by 'glib-2.0', not found
-- Found GTK3: /nix/store/6pzxmb82pda01y6zxncdmmp7lhafb75h-gtk+3-3.24.41/lib/libgtk-3.so;/nix/store/6pzxmb82pda01y6zxncdmmp7lhafb75h-gtk+3-3.24.41/lib/libgdk-3.so;/nix/store/lv6nackqis28gg7l2ic43f6nk52hb39g-zlib-1.3.1/lib/libz.so;/nix/store/7ammyzfh4i39dqxj3v0rxs21crrxp4nb-pango-1.52.2/lib/libpangocairo-1.0.so;/nix/store/7ammyzfh4i39dqxj3v0rxs21crrxp4nb-pango-1.52.2/lib/libpango-1.0.so;/nix/store/d8sw2gwfjkhigipm6jiqsgknxpjzdzrz-harfbuzz-8.4.0/lib/libharfbuzz.so;/nix/store/6rigmq2ycbpgywmq9jjyhdr6vs8k8h8x-at-spi2-core-2.52.0/lib/libatk-1.0.so;/nix/store/b9f9c4ml2k6fg0pgc80d9y4a6byqflmp-cairo-1.18.0/lib/libcairo-gobject.so;/nix/store/b9f9c4ml2k6fg0pgc80d9y4a6byqflmp-cairo-1.18.0/lib/libcairo.so;/nix/store/k2fmr13zj9rhkr29axhkaizjqn1llxaz-gdk-pixbuf-2.42.11/lib/libgdk_pixbuf-2.0.so;/nix/store/c2v6ycn0sjcpx9ww8x7j4ima6xnpssry-glib-2.80.2/lib/libgio-2.0.so;/nix/store/c2v6ycn0sjcpx9ww8x7j4ima6xnpssry-glib-2.80.2/lib/libgobject-2.0.so;/nix/store/c2v6ycn0sjcpx9ww8x7j4ima6xnpssry-glib-2.80.2/lib/libglib-2.0.so (found version "3.24.41")
-- Found OpenGL: /nix/store/dxp9ffxm50hj300xvzwhsllwgjwxvwpp-libGL-1.7.0/lib/libOpenGL.so
[0mCMake Deprecation Warning at dependencies/discord-rpc/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.

[0m
-- Could NOT find wxWidgets (missing: wxWidgets_DIR)
-- Found wxWidgets: -L/nix/store/3l863xpgr4jdf7xbcmhvvlfw7p1yxz8f-wxwidgets-3.2.4/lib;-pthread;;;-L/nix/store/jz3vvf4nsyirb25rh9dbhksm4gq6wybb-libglvnd-1.7.0/lib;-L/nix/store/1v02fxpcfm2kvy07m43msx7354aixbxa-glu-9.0.3/lib;-lwx_baseu-3.2;-lwx_gtk3u_core-3.2;-lwx_gtk3u_gl-3.2;-lwx_gtk3u_propgrid-3.2;-lwx_gtk3u_xrc-3.2 (found suitable version "3.2.4", minimum required is "3.2")
-- Checking for one of the modules 'zarchive'
-- Found ZArchive: /nix/store/lfi5z1fzxl5fyzp4ilr90gzvsbwmsq9f-zarchive-0.1.2/lib/libzarchive.a (found version "0.1.2")
-- Could NOT find libusb (missing: libusb_DIR)
-- Checking for one of the modules 'libusb-1.0;libusb'
-- Found libusb: /nix/store/s52wvz5i9nlsx0pp1z32bj5ipq5hs45l-libusb-1.0.27/lib/libusb-1.0.so (found version "1.0.27")
-- The ASM compiler identification is GNU
-- Found assembler: /nix/store/9bv7dcvmfcjnmg5mnqwqlq2wxfn8d7yi-gcc-wrapper-13.2.0/bin/gcc
-- The ASM_NASM compiler identification is NASM
-- Found assembler: /nix/store/33wcihghgnnh6xygfcjjclcspfzxz05l-nasm-2.16.03/bin/nasm
-- Configuring done (3.9s)
[31mCMake Error at src/imgui/CMakeLists.txt:15 (target_sources):
  Cannot find source file:

    ../../dependencies/imgui/imgui.cpp

[0m
[33mCMake Warning (dev) at dependencies/discord-rpc/src/CMakeLists.txt:62 (add_library):
  Policy CMP0069 is not set: INTERPROCEDURAL_OPTIMIZATION is enforced when
  enabled.  Run "cmake --help-policy CMP0069" for policy details.  Use the
  cmake_policy command to set the policy and suppress this warning.

  INTERPROCEDURAL_OPTIMIZATION property will be ignored for target
  'discord-rpc'.
This warning is for project developers.  Use -Wno-dev to suppress it.
[0m
[33mCMake Warning (dev) at dependencies/ih264d/CMakeLists.txt:5 (add_library):
  Policy CMP0069 is not set: INTERPROCEDURAL_OPTIMIZATION is enforced when
  enabled.  Run "cmake --help-policy CMP0069" for policy details.  Use the
  cmake_policy command to set the policy and suppress this warning.

  INTERPROCEDURAL_OPTIMIZATION property will be ignored for target 'ih264d'.
This warning is for project developers.  Use -Wno-dev to suppress it.
[0m
-- Generating done (0.1s)
[33mCMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    CMAKE_POLICY_DEFAULT_CMP0025
    PORTABLE

[0m
[0mCMake Generate step failed.  Build files cannot be regenerated correctly.[0m

```

### Additional context

Add any other context about the problem here.

### Notify maintainers
@zhaofengli @baduhai 
<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
output here
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/applications/emulators/cemu/default.nix
+++ b/pkgs/applications/emulators/cemu/default.nix
@@ -108,7 +108,8 @@ in stdenv.mkDerivation rec {
     tag = last (splitString "-" version);
   in ''
     rm -rf dependencies/imgui
-    ln -s ${imgui'}/include/imgui dependencies/imgui
+    # cemu expects imgui source code, not just header files
+    ln -s ${imgui'.src} dependencies/imgui
     substituteInPlace src/Common/version.h --replace " (experimental)" "-${tag} (experimental)"
     substituteInPlace dependencies/gamemode/lib/gamemode_client.h --replace "libgamemode.so.0" "${gamemode.lib}/lib/libgamemode.so.0"
   '';

